app/models/poll.rb

```diff
@@ -29,6 +29,7 @@ class Poll < ActiveRecord::Base
   after_touch       :ensure_status_is_correct
   after_create      :create_rating
   after_find        :close_if_expire
+  after_update      :draft!, if: :ready?
```

Comment:
I can't find right way to put it in earlier stage without having pain in my ass. When it strike before I should skip it in ready! action, othervise it change status to draft again and poll never cant be ready, and same about draft! action, should skip it or have endless loop, and when user vote I definatly don't wont to make poll draft and loose votation progress( but thats happend anyway and I just notice it, and callback change only status and leave votation progress untached. Bugs, bugs everywhere!)